R/fromQuantiles.R

```diff
@@ -235,8 +235,8 @@ Quantiles2LogisticNormal <- function(dosegrid,
 ##' @keywords programming
 getMinInfBeta <- function(p, q) {
   stopifnot(
```

Comment:
here we should also use `assert_probability` and avoid `stopifnot`

Reply:
Updates completed